cmd/nerdctl/build.go

```diff
@@ -103,15 +103,19 @@ func isImageSharable(buildkitHost string, namespace, uuid string) (bool, error)
 	if !ok {
 		return false, nil
 	}
-	containerdUUID, ok := labels["org.mobyproject.buildkit.worker.containerd.uuid"]
+	workerUUID, ok := labels["org.mobyproject.buildkit.worker.containerd.uuid"]
```

Comment:
This should be still containerdUUID because it is under `o.m.b.w.containerd`.

OTOH, workerSnapshotter shouldn't be containerdSnapshotter because it is not under `o.m.b.w.containerd`.